spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2206,11 +2206,12 @@ def toggle_comment(self):
 
     def comment(self):
         """Comment current line or selection."""
-        self.add_prefix(self.comment_string)
+        self.add_prefix(self.comment_string + " ")
 
     def uncomment(self):
         """Uncomment current line or selection."""
         self.remove_prefix(self.comment_string)
+        self.remove_prefix(" ")
 
     def __blockcomment_bar(self):
         return self.comment_string + '='*(79-len(self.comment_string))

```

Comment:
@Ericvulpi I think that we need to change the line above like you proposed adding a space  (`' '`) for the block comments. It should be something like:

```python
        return self.comment_string + ' ' + '=' * (78 - len(self.comment_string))
```

You can test this with `Ctrl+4` (blockcomment) and `Ctrl+5` (unblockcomment). The change should make a block comment that starts with `# ===...` instead of `#===...` (note the space between the `#` and the `=` symbols).